numerics/polynomial_in_monomial_basis_body.hpp

```diff
@@ -448,11 +447,10 @@ Primitive() const {
 
 template<typename Value_, typename Argument_, int degree_,
          template<typename, typename, int> typename Evaluator>
-template<typename, typename>
 Primitive<Value_, Argument_>
-PolynomialInMonomialBasis<Value_, Argument_, degree_, Evaluator>::
-Integrate(Argument const& argument1,
-          Argument const& argument2) const {
+PolynomialInMonomialBasis<Value_, Argument_, degree_, Evaluator>::Integrate(
+    Argument const& argument1,
+    Argument const& argument2) const requires additive_group<Value> {
```

Comment:
Let's revert the formatting change and put the `require` clause on its own line, indented by 2 positions.  Clang is not very good at formatting concepts, we should ignore it.